src/data/gqlQueries.js

```diff
@@ -167,6 +167,26 @@ export async function markMessagesAsRead(threadId) {
   return response.setThreadLastViewedAt || {}
 }
 
+export async function getEvents() {
+  const response = await gql(`{
+    meetings {
```

Comment:
Do you want to extra the meeting fields to a separate constant, and revise this to `meetings { ${meetingFields} }` like we do for things like posts' fields?

Reply:
yep, as soon as we need it in another query I will.